src/controllers/paymentController.js

```diff
@@ -0,0 +1,19 @@
+const paymentService = require("../services/paymentService");
+
+const createPaymentHistory = async (req, res) => {
+  const { orderId, paymentKey, amount } = req.body;
+
+  await paymentService.createPaymentHistory(orderId, paymentKey, amount);
+
+  return res.status(201).json({ message: "SUCCESS" });
+};
+
+const getPaymentHistory = async (req, res) => {
+  const { orderId } = req.body;
+
+  const paymentResponseData = await paymentService.getPaymentHistory(orderId);
+
+  return res.status(200).json({ message: "success", data: paymentResponseData.data });
```

Comment:
```suggestion
  return res.status(200).json({ message: "success", data: paymentResponseData });
```
`.data`를 붙여주기보다 애초에 처음부터 data에 대한 값을 받아올 수 있도록 할 수 있을 것 같습니다.